docker/datahub-ingestion-base/Dockerfile

```
@@ -31,15 +32,30 @@ RUN apt-get update && apt-get install -y \
     zip \
     unzip \
     ldap-utils \
-    openjdk-11-jre-headless \
-    && python -m pip install --upgrade pip wheel setuptools==57.5.0 \
+    && python -m pip install --no-cache --upgrade pip wheel setuptools==57.5.0 \
```

we should probably loosen the setuptools pin too
```suggestion
    && python -m pip install --no-cache --upgrade pip wheel setuptools \
```